Cue/app/actions/library.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import type { PromiseAction } from './types';
+import LibraryApi from '../api/Library';
+
+async function loadLibrary(state): PromiseAction {
+  let library = eval(await LibraryApi.fetchLibrary(state));
```

Comment:
for some reason when i was testing it last night and response.json() was returning undefined. It was late and couldnt figure out why but text() was working.

Trying it now and response.json() works so ill make the change to use that.